### PR TITLE
Update UG and fix bugs

### PR DESCRIPTION
**Pull request by teo-jun-xiong**

### CODE REVIEW

**Review by shanecsj — APPROVED** [review]

**Review by hans555 — APPROVED** [review]

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -112,147 +123,175 @@ This command allows you to view a recipe, the ingredients required, and the step
...
-``2: <STEP_2>`` +
-``.`` +
-``.`` +
-``.`` +
-``Tags: [<tag1>] [<tag2>]...``
+a|
+image::CookbookViewExample2.png[width="790"]
+
+Note: you must have added a recipe to the cookbook, otherwise Cooking Papa will not be able to show you
+the details of the recipe.
```
Comment (tzihiang) [inline docs/UserGuide.adoc:133]: Thanks for changing this long-winded explanation with an image!

**Review by tzihiang — APPROVED** [review]
LGTM! Thanks for updating the user guide!

File: src/main/java/seedu/address/logic/parser/cookbook/CookbookAddCommandParser.java [hunk 2]
```diff
@@ -52,14 +52,17 @@ public CookbookAddCommand parse(String args) throws ParseException {
         }
 
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_RECIPE_NAME, PREFIX_RECIPE_DESCRIPTION, PREFIX_INGREDIENT_NAME,
-                    PREFIX_INGREDIENT_QUANTITY, PREFIX_STEP_INDEX, PREFIX_STEP_DESCRIPTION, PREFIX_TAG);
+            ArgumentTokenizer.tokenize(args, PREFIX_RECIPE_NAME, PREFIX_RECIPE_DESCRIPTION, PREFIX_INGREDIENT_NAME,
+                PREFIX_INGREDIENT_QUANTITY, PREFIX_STEP_INDEX, PREFIX_STEP_DESCRIPTION, PREFIX_TAG);
```
Comment (gohkh) [inline src/main/java/seedu/address/logic/parser/cookbook/CookbookAddCommandParser.java]: According to the 2103 java coding style, we should have 8 spaces indentation for line wrapping, though the checkstyle script does not check for this,

**Review by gohkh — APPROVED** [review]
Thanks for checking the command messages.

**Pull request merged**